src/cpp/session/include/session/prefs/UserPrefValues.hpp

```diff
@@ -1230,6 +1232,18 @@ class UserPrefValues: public Preferences
    bool showHiddenFiles();
    core::Error setShowHiddenFiles(bool val);
 
+   /**
+    * List of files that are always shown in the Files Pane, regardless of whether hidden files are shown
```

Comment:
Should we document whether we perform a case-sensitive or case-insensitive comparison here?

Reply:
We should, done!